k8s/kube-base/kustomization.yaml

```diff
@@ -23,10 +23,18 @@ configMapGenerator:
   - name: cm-httpd-conf
     files:
       - httpd.conf
+  - name: cm-my-conf
+    files:
+      - mariadb-conf/etc-mysql-my.cnf
+  - name: cm-mariadb-conf
+    files:
+      - mariadb-conf/etc-mysql-mariadb.cnf
+  - name: cm-mysql-safe-conf
+    files:
+      - mariadb-conf/etc-mysql-conf.d-mysqld_safe_syslog.cnf
 generatorOptions:
   disableNameSuffixHash: true
```

Comment:
In general you don't want to do this: the suffix hash will allow openshift to automatically trigger a redeploy when you push an updated configmap (which otherwise won't take effect until you manually restart the relevant pod).

Reply:
In this particular case, I found it useful.

for example, If I want to deploy something that relies on the naming of a configmap separately, I don't have to constantly change the suffix in my yaml.  I had initially tried working with out disabling the name suffix hash, but that was just difficult.

I'm happy to create the configmaps manually so that we don't have to disable the name suffix hash.